src/tpm2_wrap.c

```diff
@@ -2665,6 +2745,27 @@ int wolfTPM2_NVReadAuth(WOLFTPM2_DEV* dev, WOLFTPM2_NV* nv,
         wolfTPM2_SetAuthHandle(dev, 0, &nv->handle);
     }
 
+    XMEMSET((byte*)&inPublic, 0, sizeof(inPublic));
```

Comment:
Why not use `wolfTPM2_NVReadPublic`?

Reply:
Likewise, awesome to use our own wrappers and make the code even better and more neat.